src/pages/Feed/Feed.js

```diff
@@ -1,7 +1,80 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { AiOutlineHeart, AiOutlineStar } from 'react-icons/ai';
+import { FaRegComment } from 'react-icons/fa';
+import { GrShareOption } from 'react-icons/gr';
+import * as F from './Feed.styles';
 
 function Feed() {
-  return <div>Feed</div>;
+  const [feeds, setFeeds] = useState([]);
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/feedData.json')
+      .then(res => res.json())
+      .then(data => setFeeds(data));
+  }, []);
+
+  useEffect(() => {
+    fetch('/data/rightData.json')
+      .then(res => res.json())
+      .then(data => setUsers(data));
+  });
+
+  return (
+    <F.Feed>
```

Comment:
1차 프로젝트에서는 컴포넌트의 부모 태그의 className과 컴포넌트의 이름과 동일하게 작성하라고 가이드 드렸는데
2차에서 사용하는 스타일드 컴포넌트는 스타일을 관리하는 컴포넌트지만 어쨋든 컴포넌트기 때문에 컴포넌트 이름이 동일하면
에러를 유발할 수도 있습니다. 

아래와 같은 방식으로 구현해주세요

```suggestion
    <F.FeedContainer>
```

Reply:
아 그렇군여 감사합니다.